website_medical_prescription_order_line/views/medical_prescription_order_line_template.xml

```diff
@@ -19,83 +18,60 @@
                 If you would like to order a medication, please click the green
                 button to add to cart.
             </h5>
-            <table class="table table-hover table-responsive">
-                <thead>
-                  <tr class="active">
-                      <th colspan="3">Patient</th>
-                      <th colspan="2">Rx Number</th>
-                      <th colspan="3">Medication</th>
-                      <!--<th colspan="1">Written</th>-->
-                      <th colspan="1">Remain</th>
-                      <th colspan="2">Last Disp</th>
-                      <th colspan="2">Expiration</th>
-                      <th colspan="1" class="text-right">Order</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    <t t-foreach="prescription_order_lines" t-as="line">
-                        <tr>
-                            <td colspan="3">
-                                <span t-field="line.patient_id.name" />
-                            </td>
-                            <td colspan="2" class="text-nowrap">
-                                <span t-field="line.name" />
-                            </td>
-                            <td colspan="3">
-                                <span t-field="line.medicament_id.display_name" />
-                            </td>
-                            <!--<td colspan="1">
-                                <span t-field="line.qty" />
-                            </td>-->
-                            <td colspan="1">
-                                <span t-field="line.can_dispense_qty" />
-                            </td>
-                            <td colspan="2">
+            <div class="container">
+                <t t-foreach="prescription_order_lines" t-as="line">
+                    <div class="col-xs-12 col-sm-6">
+                        <div class="panel panel-default">
+                            <div class="panel-heading">
+                                <span t-field="line.patient_id.name" /> - <span t-field="line.medicament_id.display_name" />
+                            </div>
+                            <div class="panel-body">
+                                <label class="label label-info">Rx Number</label>
+                                <span t-field="line.name" /><br/>
+
+                                <label class="label label-info">Remain</label>
+                                <span t-field="line.can_dispense_qty" /><br/>
+
                                 <t t-if="line.last_dispense_id.date_planned">
-                                   <span t-field="line.last_dispense_id.date_planned" t-field-options='{"widget": "date"}' />
+                                    <label class="label label-info">Last Disp</label><br/>
+                                    <span t-field="line.last_dispense_id.date_planned" t-field-options='{"widget": "date"}' />
```

Comment:
I don't think we want the date widget here -It's not an editable field. Using it also makes it a block element, so it's on a sep line:

![image](https://cloud.githubusercontent.com/assets/7397578/22451105/79f84990-e720-11e6-92cc-62765c9a6d4f.png)